mod scanner;

use failure;

pub fn run() -> Result<(), failure::Error> {


    Ok(())
}